go.mod

```diff
@@ -50,6 +50,7 @@ require (
 	github.com/prometheus/procfs v0.9.0 // indirect
 	github.com/rogpeppe/go-internal v1.9.0 // indirect
 	github.com/santhosh-tekuri/jsonschema/v5 v5.1.1 // indirect
+	github.com/shopspring/decimal v1.3.1
```

Comment:
Let's group this with the other direct deps

Reply:
thanks for bringing this up. will group this with other deps